packages/web-components/src/components/va-modal/va-modal-slot.css

```diff
@@ -0,0 +1,3 @@
+va-modal p:first-child {
+  margin-top: 0;
+}
```

Comment:
I could have used `::slotted(p:first-child)` with an `!important`, but I'm not sure if we want that.